api_server.py

```
@@ -28,7 +28,7 @@
 from db_store import ensure_admin_user
 from gartan_fetch import fetch_station_feed_html
 from parse_grid import parse_station_feed_html
-from utils import ensure_london, get_now
+from utils import ensure_london, get_auth_lock_info, get_now, is_auth_locked
```

Pylint (4.0.6)

[warning] Unused get_auth_lock_info imported from utils

(W0611)